packages/react-native-renderer/src/ReactNativeFiberHostComponent.js

```diff
@@ -126,7 +125,4 @@ class ReactNativeFiberHostComponent {
   }
 }
 
-// eslint-disable-next-line no-unused-expressions
-(ReactNativeFiberHostComponent.prototype: NativeMethods);
```

Comment:
These casts are actually quite important. They make sure that Paper host components and Fabric host components have the same API, and that they match the flow types that products use.

If the issue is that the now exact NativeMethods no longer matches the inexact class *HostComponent does this work?

*HostComponent: {...NativeMethods, ...}

Essentially saying explicitly that it will include NativeMethods, as well as other things.

Reply:
That doesn't work, nor does `{| ... |}` and a few other things I tried

Reply:
Maybe what I said but wrapped in $ReadOnly.

$ReadOnly<{...NativeMethods, ...}>